docs/api/system-preferences.md

```diff
@@ -426,6 +426,9 @@ This user consent was not required on macOS 10.13 High Sierra or lower so this m
 macOS 10.14 Mojave or higher requires consent for `microphone` and `camera` access.
 macOS 10.15 Catalina or higher requires consent for `screen` access.
 
+Windows 10 has a global setting controlling `microphone` and `camera` access for all win32 applications.
```

Comment:
If this is specific for win32 applications should we safety check if for APPX / Window Store builds?

Reply:
The system API returns the global value, it should work fine. I took the code from Chromium.